MetaMask/metamask-extension
# Update airgap dependencies

Upgrades metamask-airgapped-keyring and bc-ur-registry-eth

## Explanation



This PR upgrades airgap dependencies based on the following PR -- https://github.com/KeystoneHQ/keystone-airgaped-base/pull/62

Lavamoat policy file updates come from running `yarn lavamoat:auto`

## More Information


This PR adds full information to QR code sign requests. This additional information is very helpful for retrieving signatures.

## Screenshots/Screencaps



### Before


N/A

### After


N/A

## Manual Testing Steps


- Ensure metamask builds properly and no core flows are broken
- Ensure that the `address` field is set on ETHSignRequest QR Codes

## Pre-Merge Checklist
